lib/escape.rb

```diff
@@ -0,0 +1,7 @@
+module Helpers
+  def Helpers.escape_md(text)
```

Comment:
You don't need the `Helpers.` prefix on this method. 

```
module Helpers
  def escape_md(test)
  end
end
```

Would be used with `Helpers.escape_md`, or

```
include Helpers

escape_md(text)
```